types/queries/add_bookmark.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO user_bookmarked_document (document_id, user_id, bookmarked_on)
+VALUES ($1, $2, NOW());
```

Comment:
I think you might want "on conflict do nothing" at the end here. This would make sure this doesn't cause an error if a document is already bookmarked.

Also if you want, you could remove `bookmarked_on` and `NOW()` since you set that as the default right?